podpac/core/utils.py

```diff
@@ -330,4 +326,72 @@ def _get_from_url(url):
         r = None
     except RuntimeError as e:
         _log.warning("Cannot authenticate to {}. Check credentials. Error was as follows:".format(url) + str(e))
-    return r.text
+
+    return r
+
+
+def cached_property(*args, **kwargs):
+    """
+    Decorator that creates a property that is cached.
+
+    Keyword Arguments
+    -----------------
+    use_cache_ctrl : bool
+        If True, the property is cached using the Node cache_ctrl. If False, the property is only cached as a private
+        attribute. Default False.
+
+    Notes
+    -----
+    The property should not depend on any other attributes that are not tagged with ``attr=True``.
```

Comment:
This double-negative is confusing. In fact, I'm confused. What?

Reply:
Is this better?

```
Notes
-----
Podpac caching using the cache_ctrl will be unreliable if the property depends on any non-tagged traits.
The property should only use node attrs (traits tagged with ``attr=True``).
```